### DISCUSSION TITLE
Potions/objets

### DISCUSSION THREAD

**Discussion by SomniumWed** [Crownicles/Crownicles#1105] [Nouvelles idées]
Hello ! Mon idée est de pouvoir vendre/trade des potions ou des objets avec d'autres personnes (ou bien créer un shop, mais ce serait peut être un peu trop abusé...)

**Comment by Pietagorh** [comment]
Le problème c'est que ça encourage le feed
Par contre, de mémoire, y'avait un minievent où tu gagnais une copie d'une potion d'une personne que tu croises, mais a jamais été fini
